.golangci.yml

```diff
@@ -128,7 +126,6 @@ linters:
     - forbidigo #temporary, needs to be processed
     - errcheck #temporary, needs to be processed
     - dupl #temporary, needs to be processed
-    - depguard #temporary, needs to be processed
     - containedctx #temporary, needs to be processed
```

Comment:
I suggest we remove the repeated comment `#temporary, needs to be processed`, it became irrelevant.

Reply:
These linters were disalbled when @oksana-grishchenko switched this repo to single golangci-lint configuration. This should not leave like this, we should review each linter one by one and decide do we want to enable it or disable it completely. This PR is a first step.

This comments highlight that these linters disabled temporary, better to keep them for now.

Reply:
Yes, I guessed it :) How about one comment above them all?

Reply:
maybe we shouldn't enable all by default and include into configuration only ones we want to enable, what do you think?

Reply:
IMO it allows us to discover what we can use, actually a great idea to turn them on all and then disable the ones we don't want 👍

Reply:
As we agreed on the BE meetings, the initial idea was to have more strict linter config in CI - enable all linters by default and disable only the particular linters. The linter step is required in CI, which means if we enable all linters now, the CI will fail with tons of linter errors. 
So as I see this PR enables several new linters, which moves the repo forward to a more reliable and maintainable state, which is great 👍